variants/audit-logging/spec/services/auditor/event/example_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe Auditor::Event::Example do
+  subject(:event) do
+    described_class.new(
+      current_user: user,
+      remote_ip: fake_remote_ip,
+      event_info: event_specific_details,
+      timestamp: test_timestamp
+    )
+  end
+
+  it "implements the expected description" do
+    event = described_class.new(current_user: nil, remote_ip: "::1", event_info: {})
+
+    expect(event.description).to eq("The example event happened")
+  end
+end
```

Comment:
I added this test to hit code coverage requirements. I don't think a class this thin needs a test but OTOH I didn't want to exempt all these event classes from coverage because that seems like setting a bad example. Opinions particularly welcome here.